docs-markdown/src/test/suite/index-coverage.ts

```diff
@@ -34,14 +34,12 @@ function setupNyc() {
 }
 
 export function run(): Promise<void> {
-	let nyc = setupNyc();
+	// tslint:disable-next-line
+	let nyc = setupNyc(); // "const" causes coverage report error
```

Comment:
This is so confusing... I have a fix for this same issue here that does use const: https://github.com/microsoft/vscode-docs-authoring/pull/574